### ISSUE TITLE
it's this time of the year again where a new create version releases v0.5.1.j

### ISSUE THREAD

**Issue by MimzyFang** [davioliva16/create-aquatic-ambitions#12]
if you find some time to update to the new create version (v0.5.1.j) I would appreciate it

**Comment by MimzyFang** [comment]
just tried overwriting the dependency and from how it seem that might be enough

**Comment by davioliva16** [comment]
i'll work on this soon

**Comment by MarioSMB** [comment]
Just letting you know this is the case again with Create 6.0!  

`java.lang.NoClassDefFoundError: com/simibubi/create/foundation/utility/Lang`

**Comment by davioliva16** [comment]
Yeah, they changed how Fan Processing and a few classes on 0.6, currently looking into updating it
